src/frontend/mosh-server.cc

```diff
@@ -551,6 +576,8 @@ void serve( int host_fd, Terminal::Complete &terminal, ServerConnection &network
 	sel.add_fd( host_fd );
       }
 
+      network.oob()->pre_poll();
```

Comment:
I think going this other way around, would be quite a bit of (re-)work. But as long as I don't have to do it, I don't mind.